Marlin/fonts/README.fonts

```diff
@@ -1,4 +1,4 @@
-The fonts are created with Fony.exe (http://hukka.ncn.fi/?fony) because Fontforge didn't do what I want (probably lack off experience). 
+The fonts are created with Fony.exe (http://hukka.ncn.fi/?fony) because Fontforge didn't do what I want (probably lack of experience).
```

Comment:
@AnHardt -- I consider correction of spelling errors in a comment to be "cosmetic" and, as such, do not justify being processed as a separate issue. The entire point of this proposed change is just a cosmetic improvement to avoid regression of other cosmetic aspects of the code.